paper-input-error.html

```diff
@@ -43,6 +43,8 @@
         @apply(--paper-font-caption);
         @apply(--paper-input-error);
         position: absolute;
+        left:0;
+        right:0;
```

Comment:
I think this will make the error overlap `the char-counter` when it's too long:
<img width="420" alt="screen shot 2015-10-26 at 3 49 34 pm" src="https://cloud.githubusercontent.com/assets/1369170/10744716/3d3eb4c8-7bf9-11e5-9e5b-9e9c4fb1cf3c.png">

To be fair, I think this was happening before (it's happening without your patch), but it's definitely going to happen now.